web/server/entrypoint.go

```diff
@@ -249,6 +250,92 @@ func (s *robotServer) createWebOptions(cfg *config.Config) (weboptions.Options,
 	return options, nil
 }
 
+// A wrapper around actual config processing that also applies options from the
+// robot server.
+func (s *robotServer) processConfig(in *config.Config) (*config.Config, error) {
+	out, err := config.ProcessConfig(in)
+	if err != nil {
+		return nil, err
+	}
+	out.Debug = s.args.Debug || in.Debug
+	out.EnableWebProfile = s.args.WebProfile || in.EnableWebProfile
+	out.FromCommand = true
+	out.AllowInsecureCreds = s.args.AllowInsecureCreds
+	out.UntrustedEnv = s.args.UntrustedEnv
+
+	// Use ~/.viam/packages for package path if one was not specified.
```

Comment:
I added this if-check to not forcibly change the initial package path to be `~/.viam/packages` and respect whatever was unmarshaled from disk if there was anything (non-"".)